### ISSUE TITLE
After a while container will not start anymore

### ISSUE THREAD

**Issue by Franselbaer** [berngp/docker-zabbix#12]
After I did some stops and starts it'll exit immediately after starting the conainer. Had this multiple times on multiple plattforms.

I use this container often for testing purposes, or onetime tests but if i stop/start too often i can only throw away...

Best regards

**Comment by Franselbaer** [comment]
Seems to be an ubuntu networkmananger issue

**Comment by Franselbaer** [comment]
Nope. I started a new instance on a fresh ubuntu and let it run for 21 hours. The next day i stopped it and couldn't start it anymore.

**Comment by Franselbaer** [comment]
Reproduced: started a new instance, set up a cron to stop/start it every 5 minutes and after a half an hour it doesn't start anymore

**Comment by berngp** [comment]
Thanks @Franselbaer I am currently running it on a CentOS. Could you please share with me which version of Ubuntu and Docker you are using?

**Comment by Franselbaer** [comment]
Hey @berngp. Thanks for the response. I tested with ubuntu 14.10 and debian 7 with docker taken from repository (I think 1.2). Unfortunately you cannot get inside if it doesn't start anymore therefore I couldn't investigate more yet. If I find the time the next days I'll try to let the container write logs to an outside location.

**Comment by berngp** [comment]
Thank you @Franselbaer, I do my due diligence to replicate the issue. We can have mounts where the logs would be written to, I was planning on doing this anyway for MySQL too.

**Comment by berngp** [comment]
@Franselbaer the _monit_ & _AppArmor_ issue referenced in #13 was causing the syslogs to grow very quickly and might be related. Could you please run the container with the following flags.

```
--cap-add SYS_PTRACE  --security-opt apparmor:unconfined
```

**Comment by berngp** [comment]
@Franselbaer let me know if the changes above helped you out.

**Comment by meltonpineda** [comment]
Hi, I'm having the same problem. I already add the flags that you suggested but the problem still occurs. I'm running it on Centos 6.6.

**Comment by berngp** [comment]
@meltonpineda we need to move out of *monit*, is already causing several stability issues.
As refeference #22

**Comment by berngp** [comment]
@azurewraith I am migrating to `supervisord` but haven't yet finished. I'll push a labeled image that we can test out and see if `supervisord` helps out on stabilizing the container.

**Comment by azurewraith** [comment]
@berngp Thanks, that would be helpful.  I'm assuming there is a `supervisord` branch as well?

**Comment by berngp** [comment]
@azurewraith sorry for the late reply. I have the [feature/22-supervisord](/berngp/docker-zabbix/tree/feature/22-supervisord) to address the migration from `monit` to `supervisor`. It still not working but will keep track to such effort through that branch.
